include/bluetooth/gatt.h

```diff
@@ -917,6 +917,43 @@ static inline int bt_gatt_notify(struct bt_conn *conn,
 	return bt_gatt_notify_cb(conn, &params);
 }
 
+/** @brief Notify attribute value change by UUID.
+ *
+ *  Send notification of attribute value change, if connection is NULL notify
+ *  all peer that have notification enabled via CCC otherwise do a direct
+ *  notification only on the given connection.
+ *
+ *  The attribute object on the parameters is the starting point for the search
+ *  of the UUID.
+ *
+ *  @param conn	Connection object.
+ *  @param uuid	The UUID. If the server contains multiple services with the same
+ * 		UUID, then the first occurrence, starting from the attr given,
+ * 		is used.
+ *  @param attr	Pointer to an attribute that serves as the starting point for
+ * 		the search of a match for the UUID.
+ *  @param data	Pointer to Attribute data.
+ *  @param len	Attribute value length.
```

Comment:
All the tabs in this region should be replaced with spaces.

Reply:
Fixed

Reply:
Ah, yeah. Surprised that wasn't caught by the CI.

Reply:
Should be fixed now :) Thanks for catching that

Reply:
CI doesn't catch it unfortunately.